src/pages/api/venues/[venueId].ts

```diff
@@ -0,0 +1,20 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import axios from "axios";
+
+import formatResponseObject from "../../../util/api/formatResponseObject";
+import getSourceUrl from "../../../util/api/getSourceUrl";
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  const { venueId } = req.query;
+  const splitVenueId = (venueId as string).split(":");
```

Comment:
Option for code style:

```suggestion
  const [source, id] = (venueId as string).split(":");
```